src/browser-installer/chromium/revisions/linux.ts

```diff
@@ -0,0 +1,42 @@
+export default {
+    73: 625983,
+    74: 638903,
+    75: 652459,
+    76: 665032,
+    77: 681154,
+    78: 694594,
+    79: 707231,
+    80: 722374,
+    81: 737198,
+    82: 750023,
+    83: 756143,
+    84: 769125,
+    85: 782822,
+    86: 800433,
+    87: 813060,
+    88: 827143,
+    89: 843934,
+    90: 858016,
+    91: 870827,
+    92: 885357,
+    93: 902296,
+    94: 911605,
+    95: 920070,
+    96: 929514,
+    97: 938637,
+    98: 950416,
+    99: 961779,
+    100: 972803,
+    101: 982577,
+    102: 992824,
+    103: 1002974,
+    104: 1012822,
+    105: 1027072,
+    106: 1036920,
+    107: 1047812,
+    108: 1059082,
+    109: 1070158,
+    110: 1084167,
+    111: 1097778,
+    112: 1107206,
+} as Record<number, number>;
```

Comment:
I add `as Record<number, number>` so `.d.ts` would see this file as "just map" and don't save each value as constant